service-api/docker/app/Dockerfile

```diff
@@ -46,11 +46,11 @@ COPY --chown=root:root shared/docker/app/php-fpm-logging.conf /usr/local/etc/php
 COPY --chown=root:root shared/docker/app/www.conf /usr/local/etc/php-fpm.d/
 
 # Migration script
-COPY --chown=root:root service-api/docker/app/db-migrations.sh /app/db-migrations.sh
+COPY --chown=root:root service-api/docker/app/db-migrations.sh /usr/local/bin/db-migrations.sh
 COPY --chown=root:root service-api/phinx.php /app/phinx.php
 COPY --chown=root:root scripts/containers/health-check.sh /usr/local/bin/health-check.sh
 
-RUN chmod +x /app/db-migrations.sh /usr/local/bin/health-check.sh
+RUN chmod +x /usr/local/bin/db-migrations.sh /usr/local/bin/health-check.sh
```

Comment:
Why did this need to change?  No problem, just wondered

Reply:
Because the docker-compose file now mounts the entire app folder in the db-migrations.sh file was getting "overwritten" since it didn't exist at that path in the mounted folder.

This just moves it out of the way so that doesn't happen.

Reply:
👍